_posts/2017-08-13-gsoc-karl-1.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Contributing to Babel: Three GitHub Issues at a Time"
```

Comment:
Right, I put that in at the start when I was mulling over what to write for the blog post. I'll try to think of a better title for it.